debian/copyright

```diff
@@ -0,0 +1,51 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: pam_radius
+Source: https://github.com/FreeRADIUS/pam_radius
+
+Files: *
```

Comment:
That's not true... the only things from redhat / cryptocard are the `.c` and `.h` files.  Everything else has been completely changed by us.